### ISSUE TITLE
Possibility of fine tuning?

### ISSUE THREAD

**Issue by ianpurton** [coreylowman/llama-dfdx#19]
I'd like to put together a proof of concept for fine tuning large language models in rust.

My background is Rust rather than ML.

So my question is this model inference only or could I somehow use it to do training?

Would it be somehow related to the generic training loop https://github.com/coreylowman/dfdx/blob/main/examples/generic-train-loop.rs?

Thanks.

**Comment by coreylowman** [comment]
This repo is for model inference only, training would have other considerations in the modeling code like dropout, different data types (automatic mixed precision), and dataset wrangling.

I'm inclined to say we can create a separate repo for finetuning LLMs. This will let us write code more specific to training, so we don't have to abstract over both cases
